Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -544,10 +544,19 @@ Object* create_mapped_arguments_object(GlobalObject& global_object, FunctionObje
 }
 
 // 7.1.21 CanonicalNumericIndexString ( argument ), https://tc39.es/ecma262/#sec-canonicalnumericindexstring
-Value canonical_numeric_index_string(GlobalObject& global_object, Value argument)
+Value canonical_numeric_index_string(GlobalObject& global_object, PropertyName const& argument_or_number)
```

Comment:
`argument_or_number` is a weird name, it's always an argument. Let's just use the usual `property_name`.